MapleServer2/PacketHandlers/Game/SkillHandler.cs

```diff
@@ -163,6 +164,10 @@ private static void HandleAoeDamage(GameSession session, PacketReader packet)
                         session.Send("4F 00 03 E8 03 00 00 00 00 00 00 00 00 00 00 00 00 80 3F".ToByteArray());
                         session.Send("4F 00 03 D0 07 00 00 00 00 00 00 00 00 00 00 00 00 80 3F".ToByteArray());
                         session.Send("4F 00 08 01 04 01 00 00".ToByteArray());
+
```

Comment:
Have a comment that explains the purpose of this "temp fix barrier portal" or something

Reply:
its on line 162 actually :P

Reply:
my bad :D